sphinx-docs/SCREENSHOT_USAGE.MD

```diff
@@ -0,0 +1,36 @@
+Check out the ss_examples.rst file for some examples. The idea is to have a directive for specifying screenshots of the ka-lite web app that can be automagically updated in the case of UI changes or for building docs in different languages.
```

Comment:
`docs` was the old documentation directory that seems to be mostly dev related. I created `sphinx-docs` just to keep the two separate for now.